packages/NodeTypeResolver/NodeTypeResolver.php

```diff
@@ -142,6 +143,10 @@ public function getType(Node $node): Type
     {
         if ($node instanceof NullableType) {
             $type = $this->getType($node->type);
+            if ($type instanceof ObjectType && ! $type instanceof FullyQualifiedObjectType) {
+                return new MixedType();
+            }
```

Comment:
This would falsy transform all short object types to mixed.

Reply:
Fixed with `namespacedName` attribute in a `Name` node check https://github.com/rectorphp/rector-src/pull/1216/commits/9e49b535dbffbd7203101c570aedb0cd23b2e2d6 🎉